themes/wilson_theme_starterkit/templates/layout/page.html.twig

```diff
@@ -90,29 +90,30 @@
   </div>
 {% endif %}
 
-{% if page.footer_col_one or page.footer_col_two or page.footer_col_three or page.footer_col_four %}
-  <footer class="bg-grey-200">
-    <div class="container mx-auto px-5 py-16 md:flex md:flex-no-wrap">
+<footer class="bg-primary text-primary-contrast py-12">
+  <div class="container mx-auto px-5">
+    {% if page.footer_col_one or page.footer_col_two or page.footer_col_three or page.footer_col_four %}
+      <div class="footer-cols grid gap-6 md:gap-10 grid-cols-1 sm:grid-cols-2 md:grid-cols-4">
+        <div class="footer-cols__one">
+          {{ page.footer_col_one }}
+        </div>
+        <div class="footer-cols__two">
+          {{ page.footer_col_two }}
+        </div>
+        <div class="footer-cols__three">
+          {{ page.footer_col_three }}
+        </div>
+        <div class="footer-cols__four">
+          {{ page.footer_col_four }}
+        </div>
+      </div>
+    {% endif %}
 
-      {% if page.footer_col_one %}
-        {{ page.footer_col_one }}
-      {% endif %}
+    <hr class="my-8 opacity-20" />
 
-      {% if page.footer_col_two %}
-        {{ page.footer_col_two }}
-      {% endif %}
-
-      {% if page.footer_col_three %}
-        {{ page.footer_col_three }}
-      {% endif %}
-
-      {% if page.footer_col_four %}
-        {{ page.footer_col_four }}
-      {% endif %}
-
-    </div>
-  </footer>
-{% endif %}
+    <p class="text-sm md:text-right"><span class="font-medium">&copy; {{ "now"|date('Y') }} {{ site_name }}.</span> All rights reserved.</p>
```

Comment:
Would be good to use the translate filter on the copyright text `{{ 'All rights reserved.'|t }}`

Reply:
Good spot @rob-howells! I've actually made the whole copyright line translatable so that it can easily be overriden if required per region.

Reply:
Hmm, just a thought, do we want to translate the site name? A lot of time I suppose these might be a company name and these could be trademarked?

Reply:
Because the site name is a variable `{{ site_name }}` I don't think it will be translated, it'll be inserted as-is but in the right place - same as with dates. 

For example in English the correct order might be: `{{ site_name }} has a copyright for year {{ year }}` 

But in French the correct order might be: `the copyright for {{ year }} belongs to {{ site_name }}` (or it might be overridden in settings to display this way). 

`{{ year }}` will always be added as the year, and `{{ site_name }}` will always be the site name variable, but they'll appear in the right order.

If we translated the individual components - eg `{{ site_name }} . t('has a copyright for year') . {{ year }}` then you could end up with a mistranslation, because the system only translates that string in the middle (no context), or a missed translation because it's looking for the whole `site has copyright for year` string so it doesn't pick this up/it remains in the original language.

:sparkles: the more you know! (in other words - this is fine!)

Reply:
I got halfway through writing the same thing 😂

Reply:
That's interesting! And good to know for the future!